src/main/java/com/razorpay/CustomerClient.java

```diff
@@ -0,0 +1,33 @@
+package com.razorpay;
+
+import java.util.List;
+
+import org.json.JSONObject;
+
+public class CustomerClient extends ApiClient {
+
+  private String auth;
+
+  CustomerClient(String auth) {
+    super(auth);
+    this.auth = auth;
+  }
+
+  public Customer create(JSONObject request) throws RazorpayException {
+    return post(Constants.CUSTOMER_CREATE, request);  
+  }
+
+  public Customer fetch(String id) throws RazorpayException {
+    return get(String.format(Constants.CUSTOMER_GET, id), null);
+  }
+
+  public Customer edit(String id, JSONObject request) throws RazorpayException {
+    return put(String.format(Constants.CUSTOMER_EDIT, id), request);
+  }
+
+  public List<Token> fetchTokens(String id) throws RazorpayException {
+    TokenClient tokenClient = new TokenClient(auth);
+    
+    return tokenClient.fetchAll(id);
+  }
```

Comment:
Add `fetchToken(String id, String tokenId)` to this class instead of exposing TokenClient